sql/sql_table.cc

```diff
@@ -6537,8 +6537,7 @@ static bool fill_alter_inplace_info(THD *thd,
                                     ALTER_PARSER_DROP_COLUMN |
                                     ALTER_COLUMN_ORDER |
                                     ALTER_RENAME_COLUMN |
-                                    ALTER_CHANGE_COLUMN |
-                                    ALTER_COLUMN_UNVERSIONED));
+                                    ALTER_CHANGE_COLUMN));
```

Comment:
Did you test this change with other storage engines than InnoDB?

Reply:
I'll better make this for InnoDB only. Thanks for noticing.